README.md

```diff
@@ -1,2 +1,12 @@
 # techs
 An aggregation of tech blogs.
+
+## Prerequisites
+- Install nodejs
+  - `$ sudo apt install nodejs npm`
+
+## Usage
```

Comment:
I usually prefer things to be in code blocks instead:

````markdown
```console
$ npm install
$ node server.js
```
````

Usually, I also leave out `git clone` and `cd`.